source/linux/Foundational_Components/Kernel/Kernel_Drivers/Audio.rst

```diff
@@ -722,12 +722,84 @@ Potential issues
 
 .. ifconfig:: CONFIG_part_variant in ('AM62DX')
 
-   Following additional issues were observed for AM62D2-EVM in this release:
-
-   - The device requires a restart if audio is missing or plays through
-     only one channel during simultaneous playback and recording.
-   - Starting playback while recording is active introduces a small glitch
-     in the recording.
+   **To resolve audio problems during simultaneous playback and recording,
+   such as missing audio or mono output—consider these approaches:**
+
+   - Reset DACs using the below script
+
+   .. code-block:: bash
```

Comment:
@cshilwant @jeevantelukula , for future reference, this code block should be indented to fall under the list item. Otherwise the list is actually split into 2 separate lists. Also, there was no no newline separating the end of the directive from the rest of the content.